apps/hip/static/js/common.js

```diff
@@ -2,10 +2,16 @@ export function openMobileModal (target) {
   // target: element to open as a modal
   target.classList.remove("is-hidden-touch");
   document.body.classList.add('modal-is-open-hip')
+  // un-stick the navbar
+  const navbar = document.getElementById("mobile-navbar-hip");
+  navbar.classList.remove("is-sticky-hip");
 };
 
 export function closeMobileModal (target) {
-  // target: element to open as a modal
+  // target: element representing the modal to close
   target.classList.add("is-hidden-touch");
   document.body.classList.remove('modal-is-open-hip')
+  // re-stick the navbar
+  const navbar = document.getElementById("mobile-navbar-hip");
```

Comment:
I think we should change `navbar` to `header`, since there are multiple `nav` sections in the html code (header, sidebar, breadcrumbs`. Also I think we can move `const navbar = document.getElementById("mobile-navbar-hip");
` outside of the functions we are exporting and just definite it once above both function (I'm not certain about that though).

Reply:
I think there are a couple of small things we can open bug tickets for in QA that's not directly tied to this ticket.

Reply:
I'll update the name. I was afraid to move `navbar` outside the function because it looked to me like that would get imported on line 22 of `index.js` before the "DOMContentLoaded" event listener fired, so the DOM might not be built and ready yet.